### PR TITLE
b8 updates to Building Your First Web API with MVC 6

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@MikeWasson Beta8 changes + minor fixes + additional pics

### CODE REVIEW

File: mvc/getting-started/first-web-api.rst [hunk 1]
```diff
@@ -121,16 +122,14 @@ This approach makes it easier to unit test your controllers. Unit tests should i
 
 In order to inject the repository into the controller, we need to register it with the DI container. Open the *Startup.cs* file. Add the following using directive:
 
-.. literalinclude:: first-web-api/sample/src/TodoApi/Startup.cs
-    :language: c#
-    :lines: 4
+``using TodoApi.Models;``
```
Comment (danroth27) [inline mvc/getting-started/first-web-api.rst]: This should at least be a code-block with C# syntax highlighting

File: mvc/getting-started/first-web-api.rst [hunk 2]
```diff
@@ -175,14 +172,16 @@ Here is an example HTTP response for the ``GetAll`` method::
...
 Routing and URL paths 
 ^^^^^^^^^^^^^^^^^^^^^
 
 The `[HttpGet] <https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/HttpGetAttribute.cs>`_ attribute specifies that these are HTTP GET methods. The URL path for each method is constructed as follows:
 
-- Take the template string in the controller’s route attribute,  [Route("api/[controller]")]
-- Replace "[controller]" with the the name of the controller class, minus the ``Controller`` suffix. For this sample, that's "todo". 
-- If the ``[HttpGet]`` attribute also has a template string, append that to the path.
+- Take the template string in the controller’s route attribute,  ``[Route("api/[controller]")]``
+- Replace "[controller]" with the root name of the controller class ( the name of the controller class minus the ``Controller`` suffix). For this sample, the controller class name is **Todo**\Controller and the root name is "todo". ASP.NET MVC is not case sensitive.
```
Comment (danroth27) [inline mvc/getting-started/first-web-api.rst]: How about:

```
Replace "[Controller]" with the name of the controller, which is the controller class name minus the "Controller" suffix. For this sample the name of the controller is "todo" (case insensitive).
```

File: mvc/getting-started/first-web-api.rst [hunk 3]
```diff
@@ -197,9 +196,9 @@ To learn more about request routing in MVC 6, see :doc:`../controllers/routing`.
 Return values
 ^^^^^^^^^^^^^
 
-The ``GetAll`` method returns a CLR object. MVC automatically serializes the object to JSON and writes the JSON into the body of the response message. The response code for this method is 200, assuming there are no unhandled exceptions. (Unhandled exceptions are translated into 5xx errors.)
+The ``GetAll`` method returns a CLR object. MVC automatically serializes the object to `JSON <http://www.json.org/>`__ and writes the ``JSON`` into the body of the response message. The response code for this method is 200, assuming there are no unhandled exceptions. (Unhandled exceptions are translated into 5xx errors.)
```
Comment (danroth27) [inline mvc/getting-started/first-web-api.rst]: I don't think JSON needs to have the double back ticks.

File: mvc/getting-started/first-web-api.rst [hunk 4]
```diff
@@ -208,7 +207,7 @@ Use Fiddler to call the API
 ---------------------------
 
 This step is optional, but it’s useful to see the raw HTTP responses from the web API. 
-In Visual Studio, press F5 to start debugging the app. Visual Studio launches a browser and navigates to ``http://localhost:port/api/todo``, where *port* is a randomly chosen port number. If you're using Chrome, the *todo* data will be displayed. If you're using IE, IE will prompt to you open or save the *todo.json* file.
+In Visual Studio, press ^F5 to launch the app. Visual Studio launches a browser and navigates to ``http://localhost:port/api/todo``, where *port* is a randomly chosen port number. If you're using Chrome, Edge or Firefox, the *todo* data will be displayed. If you're using IE, IE will prompt to you open or save the *todo.json* file.
```
Comment (danroth27) [inline mvc/getting-started/first-web-api.rst]: Why the ^?

File: mvc/getting-started/first-web-api.rst [hunk 5]
```diff
@@ -138,18 +137,16 @@ Add a controller
...
 .. literalinclude:: first-web-api/sample/src/TodoApi/Controllers/TodoController.cs
     :language: c#
     :lines: 1-11,68-70
 
-This defines an empty controller class. In the next sections, we'll add methods to implement the API. The `[FromServices] <https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/FromServicesAttribute.cs>`_ attribute tells MVC to inject the ``ITodoRepository`` that we registered earlier.    
+This defines an empty controller class. In the next sections, we'll add methods to implement the API. The `[FromServices] <https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNet.Mvc.Core/FromServicesAttribute.cs>`_ attribute tells MVC to inject the ``ITodoRepository`` that we registered in the ``Startup`` class.    
 
 Delete the *ValuesController.cs* file from the *Controllers* folder. The project template adds it as an example controller, but we don’t need it.
  
-.. image:: first-web-api/_static/delete-values-controller.png
```
Comment (danroth27) [inline mvc/getting-started/first-web-api.rst]: Remove the delete-values-controller.png file as well.

**Comment by danroth27** [comment]
Looks good! Just a couple of minor commits. :watch:

File: mvc/getting-started/first-web-api.rst (hunk 4 above)
Comment (Rick-Anderson) [inline mvc/getting-started/first-web-api.rst]: I want them to run the app outside the debugger. We are not debugging the app.

**Comment by Rick-Anderson** [comment]
@danroth27 fixed everything.

**Comment by danroth27** [comment]
:shipit:

**Pull request merged**